tpm2tools/keys_test.go

```diff
@@ -70,3 +72,184 @@ func BenchmarkStorageRootKeyRSA(b *testing.B) {
 		key.Close()
 	}
 }
+
+func TestSeal(t *testing.T) {
+	rwc := internal.GetTPM(t)
+	defer rwc.Close()
+
+	key, err := StorageRootKeyRSA(rwc)
+	if err != nil {
+		t.Fatalf("can't create srk from template: %v", err)
+	}
+
+	secret := []byte("test")
+
+	extension := bytes.Repeat([]byte{0xAA}, sha256.Size)
+	err = tpm2.PCRExtend(rwc, tpmutil.Handle(7), tpm2.AlgSHA256, extension, "")
+	if err != nil {
+		t.Fatalf("failed to extend pcr: %v", err)
+	}
+
+	priv, pub, err := key.Seal(rwc, []int{7}, secret)
```

Comment:
Instead of having tpmutil.Handle(7) scattered about, we should define a specific test PCR at the top of this file. It should ideally be one that's unlikely to be used by another application so that testing on real hardware isn't too disruptive. I'd recommend tpmutil.Handle(15).

Reply:
Done.